main.rb

```diff
@@ -0,0 +1,149 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+def create_student(age, name)
+  print('Has Parent Permission [y/n]:')
```

Comment:
- [ ] The input option of _has parent permission_ is **throwing an error** when `y` or `n` is entered as options. Kindly ensure to improve this aspect.

**Error:**
![image](https://user-images.githubusercontent.com/12745474/144578062-608350db-6bad-4cb9-ab41-701ea49e7987.png)

Reply:
Fixed